### PR TITLE
Add lang attribute to HTML tag

### PR DESCRIPTION
**Pull request by charliepark**
This is just a small commit to make sure I have the PR flow down.

Web Content Accessibility Guidelines suggest making it easy for computers and users to know "the default human language of each Web page" — http://www.w3.org/TR/WCAG20/#meaning

### CODE REVIEW

File: public/_layout.ejs [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <!doctype html>
-<html xmlns="http://www.w3.org/1999/xhtml" xmlns:fb="http://ogp.me/ns/fb#">
+<html lang="en" xml:lang="en" xmlns="http://www.w3.org/1999/xhtml" xmlns:fb="http://ogp.me/ns/fb#">
```
Comment (alexlande) [inline public/_layout.ejs]: `xml:lang` is deprecated, we should be fine with just `lang="en"`

**Comment by davethegr8** [comment]
Per @alexlande's comment, lets drop the namespaced lang attr. Looks good otherwise.

**Comment by charliepark** [comment]
Great. Took care of that.

**Comment by davethegr8** [comment]
:+1:

**Pull request merged**